app/build.gradle

```diff
@@ -98,4 +98,14 @@ dependencies {
     implementation "io.coil-kt:coil-compose:2.2.2"
     implementation "io.coil-kt:coil-gif:2.2.2"
 
-}
\ No newline at end of file
+    // DataStore
+    implementation("androidx.datastore:datastore-preferences:1.0.0")
+    //For Coroutines if not added earlier
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4'
+    //For lifecycle mybe unnecessary
+    implementation "androidx.lifecycle:lifecycle-runtime-ktx:2.6.0-alpha03"
+    implementation "androidx.lifecycle:lifecycle-livedata-ktx:2.6.0-alpha03"
```

Comment:
We already have these dependencies in other versions (and not alpha) 😅 

I would delete adding these or the once already present